src/content/guides/installation.md

```diff
@@ -28,12 +28,17 @@ npm install --save-dev webpack
 npm install --save-dev webpack@<version>
 ```
 
-If you're using webpack 4 or later, you'll also need to install the CLI.
+If you're using webpack 4 or later, you'll also need to install the [CLI](/api/cli/).
 
 ``` bash
 npm install --save-dev webpack-cli
 ```
 
+Note that there are at least two webpack CLIs available:
```

Comment:
"at least" is not necessary here. would prefer:

_Users have a choice between two CLI packages to use:_